src/EDIT.ASM

```diff
@@ -98,10 +98,12 @@ main:
     pop cx                              ; Restore line number
     push cx                             ; Push back to stack
     mov di, dest_buffer                 ; Copy into buffer
+    add di, word [cline_length]         ; Out of the way of the clipboard

```

Comment:
The location of the buffer changes dynamically based on the length of the line in the clipboard in order to allow for lines of any size.